camera_system/coordinate_getter.py

```diff
@@ -0,0 +1,81 @@
+"""座標取得GUIツール.
+
+画像を表示し、クリックした座標を保持する
+@author miyashita64 mutotaka0426
+@note 参考: https://techacademy.jp/magazine/51035
+"""
+
+import cv2
+from typing import List
+
+
+class CoordinateGetter:
+    """画像から座標を取得するクラス.
+
+    Attributes:
+        __block_point: ブロック置き場の座標リスト
+        __base_circle: ベースサークルの座標リスト
+    """
+
+    def __init__(self, file_name: str) -> None:
+        """カメラシステムのコンストラクタ.
+
+        Args:
+            file_name (str, optional): 入力画像ファイルのパス.
+        """
+        # メンバを初期化する.
+        self.__block_point = []
+        self.__base_circle = []
+
+        # GUIツールを設定する
+        self.__img = cv2.imread(file_name)
+        self.__window_name = "Coordinate Getter"
+        # 画像を表示する.
+        self.__showImage()
+
+    def __showImage(self) -> None:
+        """画像を表示する関数."""
+        cv2.imshow(self.__window_name, self.__img)
+        cv2.setMouseCallback(self.__window_name, self.__setCoordinate)
+        cv2.waitKey(0)
+
+    def __setCoordinate(self, event, x, y, flags, params) -> None:
+        """マウス操作で取得した座標を各座標リストにセットするコールバック関数.
+
+        Args:
+            event (str, optional): イベントを示す変数.
+            x, y (int, optional): イベント発生時のマウスポインタの座標.
+            NOTE: OpenCVのコースバック関数については https://qr.paps.jp/opDWK を参照
+        """
+        # 最初の8クリックはブロック置き場の座標を取得する
+        if len(self.__block_point) < 8:
+            if event == cv2.EVENT_LBUTTONDOWN:
+                print('ブロック置き場{}：{} {}'.format(len(self.__block_point)+1, x, y))
+                self.__block_point.append([x, y])

```

Comment:
座標の値は、他のコードで間違って変更されないようにTuple型で記述するのはどうでしょうか？

```suggestion
                self.__block_point.append((x, y))
```

参考資料: [Pythonでタプルの要素を追加・変更・削除 | note.nkmk.me](https://note.nkmk.me/python-tuple-operation/#:~:text=%E6%9C%AC%E6%9D%A5%E3%80%81%E3%82%BF%E3%83%97%E3%83%AB%E3%81%AF%E6%9B%B4%E6%96%B0%E3%81%97%E3%81%9F%E3%81%8F%E3%81%AA%E3%81%84%20/%20%E3%81%95%E3%82%8C%E3%81%9F%E3%81%8F%E3%81%AA%E3%81%84%E3%83%87%E3%83%BC%E3%82%BF%E3%82%92%E8%A1%A8%E7%8F%BE%E3%81%99%E3%82%8B%E3%82%82%E3%81%AE)